cis_v110/section_5.sp

```diff
@@ -11,7 +11,10 @@ benchmark "cis_v110_5" {
     control.cis_v110_5_1,
     control.cis_v110_5_2,
   ]
-  tags          = local.cis_v110_5_common_tags
+
+  tags = merge(local.cis_v110_5_common_tags, {
+    type    = "Benchmark"
```

Comment:
Should we include the `service` tag, set to `OCI/Identity`, since all controls are for that service?